uiowa/uiowa
# Simplify env splits and prevent indexing of all drupal.uiowa.edu domains.

Resolves #1300

Resolves #1554, kinda. Purge complains without a Varnish instance running locally as does Acquia Purge. I don't think we want to set that up in DrupalVM right now, though it is possible.

### Changes
- Move memcache, robotstxt to default config enabled in all envs
- Remove unused config from env splits
- Add additional purger log channels to dev/test/prod splits
  - Potentially resolve an issue with purger config cim/cex discrepenancy.
- ~~Split robotstxt for uiowa.edu and deny all temporarily~~
  - Now handled dynamically for all sites.
- Hopefully fix acquia_connector.settings discrepancy which apparently was not resolved with the last module update.

### Testing
- Sync a site via `blt sync`.
- Ensure no config changes via `blt config:status`
- Optional: Manually send SPI data on a dev site and ensure no config changes.
- Ensure robots.txt is deny all locally and on dev
  - https://uiowa.dev.drupal.uiowa.edu/robots.txt
